src/App.js

```diff
@@ -1,27 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
```

Comment:
- [ ] Great job with your project 👏 . As stated in the project requirements all components should be `class` based this includes your `App` component. Kindly make sure `App.js` is a `class` based component 🙏 .